dazel/README.md

```diff
@@ -79,18 +81,19 @@ analyzer:
     - 'bazel-*/**'
 ```
 
-### Customizing your generated BUILD files
+# Customizing builds
```

Comment:
Maybe for another cl, but I think we need to highlight this more if its not going to be automatic. Anybody using angular will need a `build.yaml` and this sounds like a power user section.